packages/mgt-chat/src/statefulClient/chatOperationScopes.tests.ts

```diff
@@ -20,3 +20,24 @@ describe('chatOperationScopes tests', () => {
     expect(allChatScopes).to.have.members(expectedScopes);
   });
 });
+
+describe('chatListOperationScopes tests', () => {
+  it('should have a minimal permission set', () => {
+    const expectedScopes = [
+      'User.Read',
+      'User.ReadBasic.All',
+      'User.Read.All',
+      'People.Read',
```

Comment:
Are we sure some of these are duplicate permissions?

Like People.Read.All doesn't encompass People.Read? Or User.Read.All doesn't encompass User.Read and User.ReadBasic.All?

Reply:
I think they are likely to encompass, I am following mgt's convention.

Reply:
yes, User.Read.All will encompass User.Read and User.ReadBasic.All

Reply:
so they request everything? that seems odd.